postgresql/resource_postgresql_database.go

```diff
@@ -393,6 +400,10 @@ func resourcePostgreSQLDatabaseUpdate(db *DBConnection, d *schema.ResourceData)
 		return err
 	}
 
+	if err := setAlterOwnership(db, d); err != nil {
```

Comment:
This does more than setting a database "parameter", probably this function could have better name
(something like `reassignOwnership` or similar)